.gitignore

```diff
@@ -2,6 +2,8 @@
 node_modules
 dist
 
+*secret*
```

Comment:
I am using secret creds file locally to run and test GitHub actions. Added to gitignore file.